.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Good question! I copied this from the Docker GitHub Actions repo, where it said it would keep the GHA versions up to date. Having it do the same for our Dockerfiles would be great, but I think that's something to save for a future enhancement.